addons/mail/static/tests/chatter/web/follower_list_menu.test.js

```diff
@@ -203,23 +203,110 @@ test("Load 100 recipients at once", async () => {
     await openFormView("res.partner", partnerIds[0]);
-    await contains("button[title='Show Followers']", { text: "210" });
+    await contains("button[title='Show Followers']", { text: "60" });
     await click("button", { text: "Send message" });
     await contains(".o-mail-Chatter", {
-        text: "To: partner1, partner2, partner3, partner4, partner5, and 95 more",
+        text: "To: partner1, partner10, partner11, partner12, partner13, and 15 more",
     });
     await contains("button[title='Show all recipients']");
     await click("button[title='Show all recipients']");
-    await contains(".o-mail-RecipientList li", { count: 100 });
+    await contains(".o-mail-RecipientList li", { count: 20 });
     await contains(".o-mail-RecipientList", { text: "Load more" });
     await scroll(".o-mail-RecipientList", "bottom");
-    await contains(".o-mail-RecipientList li", { count: 200 });
+    await contains(".o-mail-RecipientList li", { count: 40 });
     await new Promise(setTimeout); // give enough time for the useVisible hook to register load more as hidden
     await scroll(".o-mail-RecipientList", "bottom");
-    await contains(".o-mail-RecipientList li", { count: 209 });
+    await contains(".o-mail-RecipientList li", { count: 59 });
     await contains(".o-mail-RecipientList span", { count: 0, text: "Load more" });
 });
 
+test("Load followers in sorted manner if #followers is less than 150", async () => {
+    const pyEnv = await startServer();
+    const [partnerId_1, partner_id_2, partner_id_3, partner_id_4, partner_id_5, partner_id_6] =
+        pyEnv["res.partner"].create([
+            {
+                display_name: "testuser",
+                name: "testuser",
+                email: "testuser@example.com",
+            },
+            {
+                display_name: "Testuser",
+                name: "Testuser",
+                email: "testuser1234@example.com",
+            },
+            {
+                display_name: "1Testuser",
+                name: "1Testuser",
+                email: "testuser@exam2ple.com",
+            },
+            {
+                display_name: "1testuser",
+                name: "1testuser",
+                email: "testuser23@example.com",
+            },
+            {
+                display_name: "Utestuser",
+                name: "Utestuser",
+                email: "btestuser@example.com",
+            },
+            {
+                display_name: "tsuser",
+                name: "tsuser",
+                email: "bid@example.com",
+            },
+        ]);
+    pyEnv["mail.followers"].create([
+        {
+            is_active: true,
+            partner_id: partnerId_1,
+            res_id: partnerId_1,
+            res_model: "res.partner",
+        },
+        {
+            is_active: true,
+            partner_id: partner_id_2,
+            res_id: partnerId_1,
+            res_model: "res.partner",
+        },
+        {
+            is_active: true,
```

Comment:
I'm not fond of for-of loops in test, but here the data could be made with for-of loop so the test is readable without having to scroll up/down.